src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -87,7 +98,29 @@ It supports the following flags:
 
 ### AWS specific
 
-- `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html). Only versions *11.3.1+ support this feature.
+- `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html).
+
+### OpenStack specific
+
+- `--cloud` - Name of the cloud in the `cloud-config` secret. This is almost always "openstack".
+- `--cloud-config` - Name of the `cloud-config` secret which defines the credentials for the OpenStack project in which the cluster should be created. This must be created in the organization namespace before creating a cluster.
+- `--external-network-id` - UUID of the external network to be used. Only required if multiple external networks are available.
+- `--failure-domain` - Failure domain of worker nodes.
+- `--node-cidr` - CIDR defining the IP range of cluster nodes.
+- `--node-image-uuid` - UUID of the root volume image source for the control plane and worker machines.
+- `--bastion-machine-flavor` - Flavor (a.k.a. size) of the worker node machines.
+- `--bastion-disk-size` - Size of root volumes attached to the cluster bastion machine in gigabytes. Must be greater than or equal to the size of the bastion source image (`--bastion-image-uuid`).
+- `--bastion-image-uuid` - UUID of the root volume image source for the bastion machine.
+- `--control-plane-machine-flavor` - Flavor (a.k.a. size) of the worker node machines.
+- `--control-plane-disk-size` - Size of root volumes attached to each control plane node machine in gigabytes. Must be greater than or equal to the size of the node source image.
+- `--control-plane-replicas` - Number of control plane replicas. This should be 1 for a non-HA control plane or 3 for an HA control plane (etcd requires an odd number of members).
+- `--worker-machine-flavor` - Flavor (a.k.a. size) of the worker node machines.
+- `--worker-disk-size` - Size of root volumes attached to each worker node machine in gigabytes. Must be greater than or equal to the size of the node source image (`--node-image-uuid`).
+- `--worker-replicas` - Number of replicas in the primary worker node pool.
+- `--cluster-version` (optional) - Version of `cluster-openstack` helm chart to use. If not provided, the latest version will be used.
+- `--default-apps-version` (optional) - Version of `default-apps-openstack` helm chart to use. If not provided, the latest version will be used.
+- `--dns-nameservers` (optional) - A list of DNS nameservers to be used to resolve external names.
+- `--enable-oidc` (optional) - If `true`, the control plane will be configured to accept OIDC-based authentication and Dex will be deployed in the workload cluster to authenticate with.
```

Comment:
Just verifying - does this _work_ right now?

Reply:
API server flags and apps - yes. DNS and config - no. But it might work later today. I can drop it for now.

Reply:
Removed.